editor/plugins/org.fusesource.ide.wsdl2rest/pom.xml

```diff
@@ -0,0 +1,218 @@
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.fusesource.ide.editor</groupId>
+    <artifactId>plugins</artifactId>
+    <version>10.2.0-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+  <artifactId>org.fusesource.ide.wsdl2rest</artifactId>
+  <packaging>eclipse-plugin</packaging>
+  <name>JBoss Fuse Tooling :: Camel Editor :: Plugins :: Wsdl2Rest Implemenatation </name>
+
+    <properties>
+        <version.apache.camel>2.18.0</version.apache.camel>
+        <version.apache.cxf>3.1.7</version.apache.cxf>
+        <version.apache.velocity>1.7</version.apache.velocity>
+        <version.args4j>2.0.31</version.args4j>
+        <version.javaparser>2.5.1</version.javaparser>
+        <version.javax.ws.rs.api>2.0.1</version.javax.ws.rs.api>
+        <version.junit>4.11</version.junit>
+        <version.wsdl4j>1.6.3</version.wsdl4j>
+        <version.commons.lang>2.4</version.commons.lang>
+        <version.wsdl2rest>0.7-SNAPSHOT</version.wsdl2rest>
```

Comment:
this version will be changed when a (productized?) release will be available, right?

Reply:
Hope so, yes.